### PR TITLE
allow traffic inside security group

### PR DESCRIPTION
**Pull request by kevcube**
boolean flag to enable intra-security group communication.

## why

- we create our app infrastructure beginning with the database, then we use the output security group ID that gets created by Postgres as an input in our application, to give it database access
- previously what we've done is create the database, create the app, and add the app security group to the database's inputs, but this makes circular dependencies because our app depends on other outputs of the database

### CODE REVIEW

**Comment by Gowiem** [comment]
@kevcube mind sharing where this is necessary in your PR description? That "Why" is helpful to understanding if this should be merged.

**Comment by kevcube** [comment]
@Gowiem added! I created my PR from GitHub.dev and it doesn't add any template 😵‍💫

**Review by Gowiem — APPROVED** [review]

File: variables.tf [hunk 1]
```diff
@@ -456,3 +456,9 @@ variable "subnet_group_name" {
   type        = string
   default     = ""
 }
+
+variable "allow_traffic_inside_security_group" {
```
Comment (Gowiem) [inline variables.tf:460]: @kevcube sorry to do it after I've already approved, but do you mind renaming this to `intra_secruity_group_traffic_enabled` or something similar? We do have a best practice around `bool` type variables to try and name them with the `_enabled` suffix and I almost forgot about it. 

```suggestion
variable "intra_security_group_traffic_enabled" {
```

**Comment by kevcube** [comment]
@Gowiem fixed

**Review by Gowiem — APPROVED** [review]

**Comment by Gowiem** [comment]
@kevcube Thanks for the contribution! Released as https://github.com/cloudposse/terraform-aws-rds-cluster/releases/tag/1.2.0

**Pull request merged**